mango-core/src/main/scala/org/bdgenomics/mango/models/LazyMaterialization.scala

```diff
@@ -258,8 +242,10 @@ class LazyMaterialization[T: ClassTag](sc: SparkContext, partitions: Int, chunkS
           val data = loadFromFile(reg, k)
           if (intRDD == null) {
             intRDD = IntervalRDD(data)
+            intRDD.persist(StorageLevel.MEMORY_AND_DISK)
           } else {
             intRDD = intRDD.multiput(data)
+            intRDD.persist(StorageLevel.MEMORY_AND_DISK)
```

Comment:
Are all these persist calls for test? They probably shouldnt be pushed to master